resources/views/short_url/edit.blade.php

```diff
@@ -0,0 +1,77 @@
+@extends('website.layouts.redesign.dashboard')
+
+@section('page-title')
+    Change a Short URL
+@endsection
+
+@section('container')
+
+    <div class="row justify-content-center">
+
+        <div class="col-md-4">
+
+            <div class="card mb-3">
+
+                <form method="post" action="{{ route("short_url::edit", ['id' => $url ? $url->id : 'new' ]) }}"
+                      enctype="multipart/form-data">
```

Comment:
I don't think this is necessary, as you're only submitting text

Reply:
You're right, fixed!